docs/sources/explore/_index.md

```diff
@@ -73,3 +73,7 @@ The Share shortened link capability allows you to create smaller and simpler URL
 > **Note:** Available in Grafana 8.5.0 and later versions.
 
 Enabled by default, allows users to create panels in dashboards from within Explore.
+
+### exploreMixedDatasource
+
+Disabled by default, allows users in Explore to have different datasources for different queries. If compatible, results will be combined.
```

Comment:
@gelicia why is this not a regular feature toggle??

Reply:
ah, it is, in the PR description it looked like like this
```
[exploreMixedDatasource]
enabled = true
```

which looked very strange :)

Reply:
@torkelo My apologies, I thought this format worked for feature toggling, I will use the `[feature_toggles]` consistently going forward.

Reply:
No worries :)